plugin.yaml

```diff
@@ -1,5 +1,5 @@
 name: "mapkubeapis"
-version: "0.4.1"
+version: "v0.1.0"
```

Comment:
I think we should use `0.4.1-0.1.0` to indicate we've made changes on top of 0.4.1. Easier for us to know how behind we are from upstream

@colinhoglund is that the way?

Reply:
👍 [This](https://github.com/kanopy-platform/drone/releases/tag/v2.17.0-0.1.0) is the way.

Reply:
ok, will do. I figured that we are essentially doing a hard fork so can keep our own versioning.
Another thing I wasn't sure is if the helm plugin would support the hyphen annotation but I think it should as it indicates it reads a SemVer version.